`kubeflow-dashboard` charm fails with `no such file or directory, open '//package.json'` when integrated with the rock

### Bug Description

When integrated with the charm, the following two images (the latest versions) fail with the same message:

* charmedkubeflow/centraldashboard:1.9.0-e67e71a
* charmedkubeflow/kubeflow-central-dashboard:1.8-5a3d6ea

### To Reproduce

1. Deploy CKF 1.8 or latest/edge
2. Run `juju refresh kubeflow-dashboard --resource oci-image=<image>` to refresh the used image
3. Check the container logs of the `kubeflow-dashboard`

### Environment

* Microk8s 1.25-strict/stable
* juju 3.4.3 (3.4/stable)
* CKF 1.8 (though I don't think the charm version matters)

### Relevant Log Output

```shell
unit-kubeflow-dashboard-0: 13:56:54 WARNING unit.kubeflow-dashboard/0.upgrade-charm - Start service "kubeflow-dashboard" (cannot start service: exited quickly with code 254)
unit-kubeflow-dashboard-0: 13:56:54 WARNING unit.kubeflow-dashboard/0.upgrade-charm ----- Logs from task 0 -----
unit-kubeflow-dashboard-0: 13:56:54 WARNING unit.kubeflow-dashboard/0.upgrade-charm 2024-06-19T13:56:54Z INFO Most recent service output:
unit-kubeflow-dashboard-0: 13:56:54 WARNING unit.kubeflow-dashboard/0.upgrade-charm     npm ERR! code ENOENT
unit-kubeflow-dashboard-0: 13:56:54 WARNING unit.kubeflow-dashboard/0.upgrade-charm     npm ERR! syscall open
unit-kubeflow-dashboard-0: 13:56:54 WARNING unit.kubeflow-dashboard/0.upgrade-charm     npm ERR! path //package.json
unit-kubeflow-dashboard-0: 13:56:54 WARNING unit.kubeflow-dashboard/0.upgrade-charm     npm ERR! errno -2
unit-kubeflow-dashboard-0: 13:56:54 WARNING unit.kubeflow-dashboard/0.upgrade-charm     npm ERR! enoent ENOENT: no such file or directory, open '//package.json'
unit-kubeflow-dashboard-0: 13:56:54 WARNING unit.kubeflow-dashboard/0.upgrade-charm     npm ERR! enoent This is related to npm not being able to find a file.
unit-kubeflow-dashboard-0: 13:56:54 WARNING unit.kubeflow-dashboard/0.upgrade-charm     npm ERR! enoent
unit-kubeflow-dashboard-0: 13:56:54 WARNING unit.kubeflow-dashboard/0.upgrade-charm
unit-kubeflow-dashboard-0: 13:56:54 WARNING unit.kubeflow-dashboard/0.upgrade-charm     npm ERR! A complete log of this run can be found in:
unit-kubeflow-dashboard-0: 13:56:54 WARNING unit.kubeflow-dashboard/0.upgrade-charm     npm ERR!     /root/.npm/_logs/2024-06-19T13_56_53_931Z-debug-0.log
unit-kubeflow-dashboard-0: 13:56:54 WARNING unit.kubeflow-dashboard/0.upgrade-charm 2024-06-19T13:56:54Z ERROR cannot start service: exited quickly with code 254
unit-kubeflow-dashboard-0: 13:56:54 WARNING unit.kubeflow-dashboard/0.upgrade-charm -----
unit-kubeflow-dashboard-0: 13:56:54 WARNING unit.kubeflow-dashboard/0.upgrade-charm
unit-kubeflow-dashboard-0: 13:56:54 WARNING unit.kubeflow-dashboard/0.upgrade-charm The above exception was the direct cause of the following exception:
unit-kubeflow-dashboard-0: 13:56:54 WARNING unit.kubeflow-dashboard/0.upgrade-charm
unit-kubeflow-dashboard-0: 13:56:54 WARNING unit.kubeflow-dashboard/0.upgrade-charm Traceback (most recent call last):
unit-kubeflow-dashboard-0: 13:56:54 WARNING unit.kubeflow-dashboard/0.upgrade-charm   File "./src/charm.py", line 298, in <module>
unit-kubeflow-dashboard-0: 13:56:54 WARNING unit.kubeflow-dashboard/0.upgrade-charm     main(KubeflowDashboardOperator)
unit-kubeflow-dashboard-0: 13:56:54 WARNING unit.kubeflow-dashboard/0.upgrade-charm   File "/var/lib/juju/agents/unit-kubeflow-dashboard-0/charm/venv/ops/main.py", line 439, in main
unit-kubeflow-dashboard-0: 13:56:54 WARNING unit.kubeflow-dashboard/0.upgrade-charm     framework.reemit()
unit-kubeflow-dashboard-0: 13:56:54 WARNING unit.kubeflow-dashboard/0.upgrade-charm   File "/var/lib/juju/agents/unit-kubeflow-dashboard-0/charm/venv/ops/framework.py", line 851, in reemit
```


### Additional Context

_No response_

--- a/centraldashboard/rockcraft.yaml
+++ b/centraldashboard/rockcraft.yaml
@@ -12,7 +12,7 @@ platforms:
 run-user: _daemon_
 
 services:
-  serve:
+  kubeflow-dashboard:
     override: replace
     summary: "Kubeflow central dashboard service"
     command: "/usr/bin/npm start"

